contracts/mainnet/connectors/swap-aggregator/main.sol

```diff
@@ -0,0 +1,95 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "./helpers.sol";
+import "./events.sol";
+
+contract SwapAggregatorConnector is Helpers, Events {
+    struct RouterInfo {
+        address router;
+        address allowanceHolder;
+    }
+
+    mapping(string => RouterInfo) public routers;
+
+    address public chief;
+    address public master;
+
+    modifier onlyChiefOrMaster() {
+        require(msg.sender == chief || msg.sender == master, "Not authorized");
```

Comment:
@SamarendraGouda You need to check if an msg.sender is chief or not from connectorsV2 contract. And master from instaIndex contract.